tests/MimeContentTypeTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+
+namespace Mmoreram\React\Tests;
+
+use PHPUnit\Framework\TestCase;
+use React\EventLoop\Factory;
+use Mmoreram\React;
+use Clue\React\Block;
+
+/**
+ * Class MimeContentTypeTest
+ */
+class MimeContentTypeTest extends TestCase
+{
+    function testIt()
+    {
+        $loop = Factory::create();
+        $promises = [];
+        $promises[] = React\mime_content_type(__DIR__ . '/static/file1.txt', $loop);
+        $promises[] = React\mime_content_type(__DIR__ . '/static/pixel.png', $loop);
+
+        $results = Block\awaitAll($promises, $loop);
+        $this->assertEquals('text/plain', $results[0]);
+        $this->assertEquals('image/png', $results[1]);
+    }
+
+    function testRejections()
+    {
+        $loop = Factory::create();
+        $promises = [];
+        $promises[] = React\mime_content_type(__DIR__ . '/static/nonexisting.txt', $loop);
+
+        foreach ($promises as $promise) {
+            try {
+                Block\await($promise, $loop);
+                $this->fail('Runtime exception should be thrown here');
```

Comment:
You have a bug here. You throw an exception and then immediately catch it. You test will always work, even if you promise resolves.

Reply:
I don't get it. This test will work only it an Exception is thrown here. Otherwise, if the promise is resolved, we will have the `$this->fail()`.

Reply:
Try to print an exception that you catch. It says: `Runtime exception should be thrown here`.

`$this->failI()` under the hood throws an exception, an right in the catch block you catch it.

Reply:
Hum! You're totally right. Fixed